x-pack/performance/journeys/cloud_security_dashboard.ts

```diff
@@ -48,6 +48,10 @@ export const journey = new Journey({
   //   maxDuration: '10m',
   // },
 }).step('Go to cloud security dashboards Page', async ({ page, kbnUrl }) => {
-  await page.goto(kbnUrl.get(`/app/security/cloud_security_posture/dashboard`));
-  await page.waitForSelector(subj('csp:dashboard-sections-table-header-score'));
+  const shouldSkip = true;
+
+  if (!shouldSkip) {
+    await page.goto(kbnUrl.get(`/app/security/cloud_security_posture/dashboard`));
+    await page.waitForSelector(subj('csp:dashboard-sections-table-header-score'));
+  }
```

Comment:
Could you explain this change? If the step is redundant, I suggest to remove it.
But if it is temporary fix, adding a comment would be a good thing.

Reply:
I was attempting to skip the test. I will just comment out the code and add a comment! Thank you.